README.md

```diff
@@ -127,6 +127,26 @@ Custom registries are not currently supported
 ([#160](https://github.com/obi1kenobi/cargo-semver-checks/issues/160)), so crates published on
 registries other than crates.io should use one of the other approaches of generating the baseline.
 
+#### Git repository detection
+
+When looking up a git revision with `--baseline-rev`,
+`cargo-semver-checks` will walk up the current directory until it finds a `.git/` directory to resolve the revision and extract the corresponding worktree.
+You can use the following environment variables to influence the git repository detection
+
+- [`GIT_CEILING_DIRECTORIES`] to stop automatic git repository discovery when reaching certain directories.
+- [`GIT_DIR`] to explicitly set the location of the `.git/` directory.
```

Comment:
Does this have to be an absolute path, or would a relative path do? If relative, is it relative to the `pwd` or something else?

Also, what's the default value / behavior here?